tools/preprocess_frozen_modules.py

```diff
@@ -35,6 +35,7 @@ def copy_and_process(in_dir, out_dir):
         # Skip library examples directory and subfolders.
         relative_path_parts = Path(root).relative_to(in_dir).parts
         if relative_path_parts and relative_path_parts[0] in ['examples', 'docs', 'tests']:
+            del subdirs[:]
             continue
```

Comment:
I'm confused: the `del subdirs[:]` should not be necessary.T he `continue` is meant to just skip the entire unwanted subdir.  I added a `print(file)` after the `if file.endswith(".py")` on line 49, and I don't see that the example files are being copied and frozen.  If you look in `/build-circuitplayground_express/frozen_mpy` you'll also see which files are compiled to `.mpy` and the example directories are not there even without this change.

Reply:
OK, got it. it's subdirs of `subdirs` that we are worried about.